core/amber/src/main/resources/python_udf/topic_modeling_mallet_trainer.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+# Use gensim 3.8.3
+import gensim
+import gensim.corpora as corpora
+import pandas
+import os
+
+import texera_udf_operator_base
+
+
+class TopicModeling(texera_udf_operator_base.TexeraBlockingUnsupervisedTrainerOperator):
+    logger = logging.getLogger("PythonUDF.TopicModelingTrainer")
+
+    @texera_udf_operator_base.exception(logger)
+    def open(self, *args):
+        super(TopicModeling, self).open(*args)
+
+        # TODO: _train_args from user input args
+        if len(args) >= 3:
+            MALLET_HOME = str(args[1])
+            NUM_TOPICS = int(args[2])
+        else:
+            self.logger.exception("Not enough arguments in topic modeling operator")
+            raise RuntimeError("Not enough arguments in topic modeling operator")
```

Comment:
please remove the exception log here as it will be duplicated. 

The raised exception will be caught and log errors correctly in the handler.

Reply:
sure